### PR TITLE
[MRG] allow for n/a to be a valid date in acq_time

### LINKED ISSUE #538: acq_time should  allow for n/a but currently it doesn't
since the recent work on `acq_time`, we get issues when trying to validate an `acq_time` that is set to `n/a`, see: https://github.com/mne-tools/mne-bids/issues/71#issuecomment-417441638

`n/a` should be accepted as valid, probably to be done here:

https://github.com/INCF/bids-validator/blob/db7415dbd8482077bddcf13edb2f6dd4905559e6/validators/tsv.js#L320-L340

### PR DESCRIPTION
**Pull request by sappelhoff**
fixes #538

### CODE REVIEW

**Comment by chrisgorgo** [comment]
Could you modify or add a test covering this case? Thanks!

**Comment by sappelhoff** [comment]
also fixes https://github.com/mne-tools/mne-bids/issues/71#issuecomment-417441638

**Comment by jasmainak** [comment]
Thanks @sappelhoff for the PR !

File: validators/tsv.js [hunk 1]
```diff
@@ -326,7 +326,10 @@ const checkAcqTimeFormat = function(rows, file, issues) {
     const line = testRows[i]
     const lineValues = line.trim().split('\t')
     const acqTime = lineValues[acqTimeColumn]
-    const isValid = dateIsValid(parseDate(acqTime, format, new Date()))
+    var isValid = dateIsValid(parseDate(acqTime, format, new Date()))
+    if (acqTime === 'n/a') {
```
Comment (jasmainak) [inline validators/tsv.js:330]: if `acqTime` is `'n/a'` why does `parseDate` not throw an error?

Comment (sappelhoff) [reply validators/tsv.js:330]: it returns `Invalid Date` if the parsing according to `format` fails

**Comment by chrisgorgo** [comment]
Thanks!

**Pull request merged**